opt/prepare-aiidalab.sh

```diff
@@ -66,7 +78,7 @@ if [ ! -e /home/$SYSTEM_USER/apps ]; then
    touch /home/$SYSTEM_USER/apps/__init__.py
    git clone https://github.com/aiidalab/aiidalab-home /home/$SYSTEM_USER/apps/home
    cd /home/$SYSTEM_USER/apps/home
-   git checkout aiida_v1.0
+   git checkout develop
```

Comment:
Could you make a note to change this in the future?

I guess the proper branch for the container will be `master`, once the migration has been completed.

Reply:
Made an issue for that: https://github.com/aiidalab/aiidalab-docker-stack/issues/38